estela-web/src/components/SpiderDetailPage/index.tsx

```diff
@@ -149,8 +245,325 @@ export class SpiderDetailPage extends Component<RouteComponentProps<RouteParams>
         });
     };
 
+    onChangeStatus = (index: number, count: number) => {
+        if (count === 0) {
+            const tableStatus = this.state.tableStatus;
+            tableStatus[index] = !tableStatus[index];
+            this.setState({ tableStatus: tableStatus });
+        }
+    };
+
+    overview = (): React.ReactNode => {
+        const { tableStatus, jobs, count, current, queueJobs, runningJobs, completedJobs, lastDeployDate } = this.state;
+        return (
+            <Content className="my-4">
+                <Row className="my-6 grid grid-cols-4 text-base h-full">
+                    <Layout className="bg-metal col-span-1 h-44">
+                        <Content className="white-background mr-5 p-3 rounded-lg">
+                            <p className="text-base text-silver p-2">SCHEDULED JOBS</p>
+                            <p className="text-xl font-bold p-2 leading-8">{/*To implement*/}</p>
+                        </Content>
+                    </Layout>
+                    <Layout className="bg-metal col-span-1 h-44">
+                        <Content className="white-background mr-5 p-3 rounded-lg">
+                            <p className="text-base text-silver p-2">JOBS</p>
+                            <p className="text-xl font-bold p-2 leading-8">{jobs.length}</p>
+                        </Content>
```

Comment:
It's possible to get the cronjobs check every jobs if belong a cronjob

Reply:
Done, filtering all cronjobs that are not null associated to all the jobs and then applying "unique" operation with functional array methods.